src/investarena/components/CreatePostForecast/CreatePostForecast.js

```diff
@@ -316,11 +323,17 @@ class CreatePostForecast extends Component {
                       dropdownClassName="st-create-post-select__dropDown"
                       showSearch
                     >
-                      {optionsQuote.map(option => (
-                        <Select.Option key={option.value} value={option.value} title={option.label}>
-                          {option.label}
-                        </Select.Option>
-                      ))}
+                      {optionsQuote.map(option => {
+                        const label =
+                          platformName === 'beaxy'
+                            ? toolLabelParser(option.label, currencySettings)
```

Comment:
Complicated solution. It's better to change selector to get optionsQuote in proper format